### ISSUE TITLE
How would I go about hacking another spine into a slot.

### ISSUE THREAD

**Issue by Billydwilliams** [pixijs-userland/spine#226]
For example I have a house with a slot named 'people' and I want to be able to replace them with another spine (I know you can with a external texture but I want them to have their own animations). The layer order is important so they actually look like theyre in the house.  Id really like to avoid splitting up the house into two (or more) spines.  Is this possible? If so, I'm not sure where to begin.

**Comment by ivanpopelyshev** [comment]
Add it to slot container.

```js
var container = spine.slotContainers[spine.skeleton.findSlotIndex("blabla")]
```

**Comment by Billydwilliams** [comment]
This helped a lot but I have a weird issue where the slot has no rotation or anything applied but once i put it in that container it displays at an odd angle and looks to be flipped across the x axis even though its scale is 1.

**Comment by ivanpopelyshev** [comment]
sometimes it uses different type of transforms, TransformBase, and you really cant affect it if you not patch `Spine.ts` file.

**Comment by ivanpopelyshev** [comment]
Try override `transformHack`: https://github.com/pixijs/pixi-spine/blob/master/src/Spine.ts#L645 , its 1 by default, try to set 2 there for your spine objects.

**Comment by Billydwilliams** [comment]
I'll give it a try sometime this week. Thanks for the help ivan. Always a pleasure!

**Comment by Billydwilliams** [comment]
This may sound weird but, is there a way for the width and height of a child to not count towards the parents width and height value?

**Comment by ivanpopelyshev** [comment]
Nope. That's how width/height work in Flash and Pixi. you have to override getLocalBounds and getBounds to get bounding box from Spine, please look at other discussions in this repo issues.

**Comment by Billydwilliams** [comment]
kk will do thanks!
